fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -185,6 +196,8 @@ struct TenantManagementWorkload : TestWorkload {
 			    self->mvDb, self->dataClusterName, g_simulator->extraDatabases[0], entry));
 		}
 
+		// set/load test parameters from metacluster
+		std::cout << "Communicate test parameters...\n";
```

Comment:
```suggestion
		// set/load test parameters from metacluster
```

Reply:
This one is a little bit different because it will be logged by all the clients. If you want to keep it, my suggestion would be to log it only for client 0 or to log something more specific for each client, including the client ID. Something like:

```
fmt::print("Sending test parameters to other clients\n");
```

and possibly

```
fmt::print("Loading test parameters on client {}\n", self->clientId);
```